BrickWall/models.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+from flask import Flask, render_template, url_for, flash, redirect
+from flask_sqlalchemy import SQLAlchemy
+#from forms import RegistrationForm, LoginForm
+app = Flask(__name__)
+
+app.config['SECRET_KEY'] = 'thisIsATestSecretKey3'
+app.config['SQLALCHEMY_DATABASE_URL'] = 'sqlite:///site.db'
+db = SQLAlchemy(app)
+
+
+class User(db.model):
+    id = db.Column(db.Integer, primary_key = True)
+    username = db.Column(db.String(20), unique=True, nullable=False)
+    email = db.Column(db.String(120), unique=True, nullable=False)
+    image_file = db.Column(db.String(20), nullable=False, default='default.jpg')
+    password = db.Column(db.String(60), nullable=False)
+    posts = db.relationship('Data', backref='author', lazy=True)
+
+    def __repr__(self):
+        return f"User('{self.username}', '{self.email}', '{self.image_file}')"
+
+
+class Data(db.model):
```

Comment:
This isn't descriptive to our model at all. Please look at #4 and make 3 models that follow the schema provided